react/features/base/jwt/constants.ts

```diff
@@ -20,10 +20,10 @@ export const MEET_FEATURES = {
 
 /**
  * A mapping between jwt features and toolbar buttons keys.
+ * We don't need recording in here, as it will disable and local recording.
```

Comment:
Not sure I understand the message. Can you elaborate?

Reply:
Cause even when feature recording(the feature is for server-side recording) is disabled you see local recording.

Reply:
Or I'm wrong ... ?